### DISCUSSION TITLE
Master branch feature contribution

### DISCUSSION THREAD

**Discussion by stephencarr** [SynthstromAudible/DelugeFirmware#13] [General]
What is the plan for users contributing to master? I can't see how it will work with gatekeeping if there is not a consensus around what needs to be built/what are we trying to build. 

I think lots of exciting forks will be spun off,  but I'm unclear about what the plans are for this master repo.

**Comment by ian-jorgensen** [comment]
> What is the plan for users contributing to master? I can't see how it will work with gatekeeping if there is not a consensus around what needs to be built/what are we trying to build.
> 
> I think lots of exciting forks will be spun off, but I'm unclear about what the plans are for this master repo.

cool. We can talk about this in the Q&A, just to clarify though, do you mean the 'master' as in the official software supplied with all Deluges sold by Synthstrom and distributed from our website, or do you mean the primary 'Community' branch?

**Comment by weavermedia** [comment]
@ian-jorgensen I have the same question and in my case I mean this community branch.

**Reply by ian-jorgensen** [reply]
I think it's going to take a wee while for this to become clearer. Things are moving really fast and I guess key thing is that developments that use key UI are thoughtfully considered/voted on before being implemented. The community firmware does need to be at least pretty stable - compared to the experimental releases and its going to do peoples heads in if UI keeps changing for the community release.
We (Synthstrom) are trying to stay a little hands off with this tbh and let the community work it out. We've got people helping with PR's and general support, but I felt it wouldn't be that fair for us to also have people setting up strategy etc, at that point its more or less us and no point to this all, haha. But I guess it might pay for someone to take a bit more of a 'strategy' role and create some documents which outline all the avail UI - slotting in possible features into that and making sure nothing is given up too hastily.
Create communication methods for the community to feedback directly to the dev's. There is lots that can be done in that regards, just needs some peeps with time and desire to manage that sort of planning.
Also, the creator has to be wiling to give their code back to the community. Lots of things need to come together and it might take a little while for things to get clearer.
